Source/WebCore/bindings/js/ScriptController.cpp

```diff
@@ -824,16 +825,24 @@ void ScriptController::executeJavaScriptURL(const URL& url, RefPtr<SecurityOrigi
     if (requesterSecurityOrigin && !requesterSecurityOrigin->isSameOriginDomain(ownerDocument->securityOrigin()))
         return;
 
-    if (!frame->page() || !ownerDocument->checkedContentSecurityPolicy()->allowJavaScriptURLs(ownerDocument->url().string(), eventHandlerPosition().m_line, url.string(), nullptr))
+    if (!frame->page())
         return;
 
-    const int javascriptSchemeLength = sizeof("javascript:") - 1;
-
     JSDOMGlobalObject* globalObject = jsWindowProxy(mainThreadNormalWorld()).window();
+
+    String urlString(url.string());
```

Comment:
It needs to be a local variable because it can be mutated by the preNavigationCheck and then is passed to allowJavaScriptURLs.

Based on https://w3c.github.io/trusted-types/dist/spec/#require-trusted-types-for-pre-navigation-check the URL needs to be changed for this later check.